content/en/docs/tasks/observability/distributed-tracing/configurability/index.md

```diff
@@ -27,6 +27,35 @@ EOF
 
 Then, you can append any configuration options to the `tracing.yaml` file.
 
+## Add `istio.proxy.io/config` annotation to your Pod metadata specification
```

Comment:
(here and elsewhere): shouldn't this be `proxy.istio.io/config` ?

Reply:
addressed